osu.Game/Input/KeyBindingStore.cs

```diff
@@ -37,7 +37,7 @@ public KeyBindingStore(DatabaseContextFactory contextFactory, RulesetStore rules
         /// </summary>
         /// <param name="globalAction">The action to lookup.</param>
         /// <returns>A set of display strings for all the user's key configuration for the action.</returns>
-        public IEnumerable<string> GetReadableKeyCombinationsFor(GlobalAction globalAction)
+        public IEnumerable<string> GetReadableKeyCombinationsFor(GlobalAction? globalAction)
```

Comment:
I don't think you need to change this. Just only do the lookup when the `GlobalAction` is non-null (in `MusicKeyBindingHandler`).